core/src/test/java/feign/CustomPojo.java

```diff
@@ -22,4 +22,12 @@
     this.name = name;
     this.number = number;
   }
+
+  public String getName() {
```

Comment:
@kdavisk6 this seems a breaking change to me.

Before `feign` would read fields... with this change, you need to have getters/setters